VueJS/index.js

```diff
@@ -1,13 +1,33 @@
 import List from './components/List.js'
 import ListItem from './components/ListItem.js'
 
+Vue.component('todo-list', List);
```

Comment:
I don't know if this is really necessary. :thinking:  At least it would be better of in `VueJS/components/List.js`